packages/components-react/tests/vrt/specs/segmented-control.vrt.ts

```diff
@@ -1,5 +1,5 @@
 import { defaultViewports, getVisualRegressionTester, vrtTest } from '@porsche-design-system/shared/testing';
 
-xit.each(defaultViewports)('should have no visual regression for viewport %s', async (viewport) => {
+it.each(defaultViewports)('should have no visual regression for viewport %s', async (viewport) => {
```

Comment:
Scaled component VRT test is missing in `packages/components-js/tests/vrt/puppeteer/specs/component-scaling.vrt.ts`.

Reply:
Added Scaled component VRT test